crates/trigger-http/src/spin.rs

```diff
@@ -27,11 +27,27 @@ impl HttpExecutor for SpinHttpExecutor {
             component_id
         );
 
-        let (instance, store) = engine.prepare_instance(component_id).await?;
+        let (instance, mut store) = engine.prepare_instance(component_id).await?;
         let EitherInstance::Component(instance) = instance else {
             unreachable!()
         };
 
+        if let Some(authority) = req.uri().authority() {
+            if let Some(scheme) = req.uri().scheme_str() {
+                if let Some(outbound_http_handle) = engine
+                .engine
+                .find_host_component_handle::<std::sync::Arc<outbound_http::OutboundHttpComponent>>(
+                )
```

Comment:
Oh rustfmt no

(no action for you, I am just sad at the tool)

Reply:
I don't think `rustfmt` produced this, I think it gave up on this code for some reason...

Reply:
Looks like it gave up because the function call was too long. If you add `use`s for `Arc` and `OutboundHttpComponent` it will happily reformat this.

Reply:
good tip! Moving it to its own function helped but I added the uses anyway for readability.